src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        System.out.println(lottery.getRandomBall());
```

Comment:
Please read common mistakes

Reply:
Thank you, it turns out I didn't fully understand the comment from 'yevheniisynytsia', but now I got it, it's corrected.